packages/trader/src/AppV2/Components/StopLoss/stop-loss.tsx

```diff
@@ -17,6 +23,7 @@ const StopLoss = observer(() => {
                 <Localize i18n_default_text='When your loss reaches or exceeds the set amount, your trade will be closed automatically.' />
             }
             value={limit_order?.stop_loss?.order_amount}
+            validation_params={validation_params}
```

Comment:
Please, check https://github.com/deriv-com/deriv-app/pull/17321/files#r1816145645
Maybe we can avoid prop drilling

Reply:
Sure, it shouldn't be needed if I get this from the hook in risk component